### ISSUE TITLE
Error running tests: failed to wait for mysql command: exit status 1

### ISSUE THREAD

**Issue by lyoshenka** [volatiletech/sqlboiler#182]
## The Problem

Running `go test ./model` returns the error 

```
exit status 1
Unable to execute setup: failed to wait for mysql command: exit status 1
```

## Things I Tried

As suggested in https://github.com/volatiletech/sqlboiler/issues/95, I tried running `mysqldump` to make sure it has the right permissions. I also tried `mysql --defaults-file=/tmp/optionfile013832393` (the command that seems to cause the error) to make sure the config file was being read correctly (it was). I can generate the models without any trouble. 

I'm not sure what else to try.

## Details

- Go v1.8.1
- SQLBoiler v2.4.0
- mysql  v5.7.18

**Comment by aarondl** [comment]
It's difficult to debug this without knowing a little bit more. Most of these issues come down to mariadb compatibility issues or permissions. The other thing you could try is to get the stdout/stderr from the mysql process in Go by modifying the generated tests. If you create yet another pipe around the process when that command is being called and point it to `os.Stdout` you might get some output that will tell you what's happening in more detail. This would be my next step.

Unfortunately without more details here I won't be able to help further. If you lack the knowledge on how to do what I suggested above I might be able to provide you with a patch. Let me know.

**Comment by lyoshenka** [comment]
You were right, its a bug in Percona 5.7.18-15, probably due to porting Facebook's MySQL MyRocks engine to Percona Server 5.7. It's fixed in 5.7.18-16 (latest release at the moment).

Thanks for your help.
